WordPress/Classes/ViewRelated/Reader/ReaderDetailCoordinator.swift

```diff
@@ -124,6 +145,26 @@ class ReaderDetailCoordinator {
         })
     }
 
+    /// Requests a ReaderPost from the service and updates the View.
+    ///
+    /// Use this method to fetch a ReaderPost from a URL.
+    /// - Parameter url: a post URL
+    private func fetch(_ url: URL) {
+        service.fetchPost(at: postURL,
+                          success: { [weak self] post in
+                            guard let post = post else {
```

Comment:
Why do we need this guard here? 
self?.post is optional, there's a guard in renderPostAndBumpStats and it could be `self?.view?.show(title: post?.postTitle)`

Reply:
Changed in 0c01b80b1702ad75995e008bc63241b40c0757a4

Thanks for pointing that out!